backend/db/mongo.go

```diff
@@ -41,8 +40,11 @@ func (c *Database) insertArticle(article Article) (string, error) {
 // 	return User{}, nil
 // }
 
-// fetch all Article from mongo
-func (c *Database) FindAllArticles() ([]Article, error) {
+
+
+// fetch ALL Article from mongo if numArticles == -1 (FLAG)
```

Comment:
I appreciate the comments! Let's clean them up (we don't need the unnecessary capitalization, and it can be in plain English) once we've refactored :)

Hopefully, we can also later document all our exported functions this way for go doc :)